st2api/st2api/controllers/v1/packs.py

```diff
@@ -284,6 +289,10 @@ class PacksController(BasePacksController):
     views = PackViewsController()
     index = PacksIndexController()
 
+    def __init__(self):
+        super(PacksController, self).__init__()
+        self.get_one_db_method = self._get_by_ref_or_id
```

Comment:
How about we use another decorator? My point, if that's the problem with RBAC, we need to fix it RBAC side. If we have enough RBAC problems, we should rewrite it. We would need to rewrite it anyway for st2api v2 so we better start gathering what's wrong with it.